js/layout-javascript/news-feed-code.js

```diff
@@ -157,6 +157,37 @@ DynamicList.prototype.attachObservers = function() {
         console.error(error);
       });
     })
+    .on('click', '.sort-group .list-sort li', function (e) {
+      e.stopPropagation();
+
+      var $sortListItem = $(e.currentTarget);
+      var oldSortOrder = $sortListItem.data('sortOrder');
+      var sortOrder;
+      var sortField = $sortListItem.data('sortField');;
+      var $sortOrderIcon = $sortListItem.find('i');
+      var $sortList = _this.$container.find('.sort-list-news li');
+      var sortClasses = {
+        'none': 'fa-sort',
+        'asc': 'fa-sort-asc',
+        'desc': 'fa-sort-desc'
+      };
+      var newSortOrder = {
+        'none': 'asc',
+        'asc': 'desc',
+        'desc': 'asc'
+      };
+
+      sortOrder = newSortOrder[oldSortOrder];
+      _this.Utils.DOM.resetSortIcons({ $sortList: $sortList });
+
+      $sortOrderIcon.removeClass(_.values(sortClasses).join(' ')).addClass(sortClasses[sortOrder]);
+      $sortListItem.data('sortOrder', sortOrder);
```

Comment:
```js
      _this.Utils.DOM.resetSortIcons({ $sortList: $sortList });

      $sortOrderIcon.removeClass(_.values(sortClasses).join(' ')).addClass(sortClasses[sortOrder]);
      $sortListItem.data('sortOrder', sortOrder);
```

These 3 lines should be part of `searchData()`. This way, whenever `searchData()` is executed with valid `sortOrder` and `sortField`, the UI is updated accordingly.